docker-compose-example.yml

```diff
@@ -11,6 +11,7 @@ services:
             ENABLE_ENV_FILE: 1
             ENABLE_LOCALCONF: 1
             API_TOKEN: "<YOUR GITHUB API TOKEN>"
+            COOKIE_VALIDATION_KEY: "<YOUR COOKIE VALIDATION KEY>"
```

Comment:
This variable is already set in `.env`. We don't want to duplicate all app configuration vars in the dockerfile.